db/schema.rb

```diff
@@ -997,8 +992,13 @@
     t.datetime "reset_password_sent_at"
     t.string   "api_key",                :limit => 40
     t.integer  "enterprise_limit",                     :default => 1, :null => false
+    t.string   "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string   "unconfirmed_email"
```

Comment:
That's really weird.  I just added a single migration to remove the enterprise confirmation columns and ran `rake db:migrate`. Maybe I need to fetch from upstream and rebase...